### DISCUSSION TITLE
How to add flags to forced processes?

### DISCUSSION THREAD

**Discussion by ghost** [sandboxie-plus/Sandboxie#3726] [Q&A Feedback]
I want applications to launch Chrome.exe as default web browser, but with specific flags that are normally added to shortcuts of executable files. I can create a shortcut to launch Chrome.exe (from explorer) in Sandboxie with specified flags, but other apps are going to launch only Chrome.exe. Flags must be added somewhere to make Sandboxie apply them to whichever executable files by default.

**Comment by ghost** [comment]
Bump to this one! For example, Explorer.exe needs to run with the following parameters to work correctly in Sandboxie - "C:\Windows\explorer.exe /e,::{20D04FE0-3AEA-1069-A2D8-08002B30309D}" . How can I use that for ForceProcess? If I set ForceProcess=Explorer.exe then it is not going to work correctly.

**Comment by offhub** [comment]
You can't add it because there is no command line check. Also, it doesn't make sense to set `explorer.exe` as a forced process.

**Comment by ghost** [comment]
I think launching Forced programs with desired flags would be a good feature to add.
